commercial/app/controllers/PrebidAnalyticsController.scala

```diff
@@ -0,0 +1,30 @@
+package commercial.controllers
+
+import common.Logging
+import conf.switches.Switches.prebidAnalytics
+import model.Cached.WithoutRevalidationResult
+import model.{CacheTime, Cached, TinyResponse}
+import play.api.libs.json.JsValue
+import play.api.libs.json.Json.prettyPrint
+import play.api.mvc._
+
+class PrebidAnalyticsController(val controllerComponents: ControllerComponents) extends BaseController with Logging {
+
+  private def serve404[A](implicit request: Request[A]) =
+    Cached(CacheTime.NotFound)(WithoutRevalidationResult(NotFound))
```

Comment:
Why is this using a `WithoutRevalidationResult` and not a `RevalidatableResult`?

Reply:
I don't really know what it means.  I just pinched it.  I didn't want it to be revalidated if the switch was off.  So it's meant to be pessimistic.